### PR TITLE
moving site collection termstore APIs from beta to v1.0

### PR DESCRIPTION
**Pull request by vishriv**

### CODE REVIEW

File: changelog/Microsoft.SharePoint.json [hunk 1]
```diff
@@ -1,5 +1,23 @@
 {
   "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "f008d427-4aa4-452d-aa93-0c5c623fdfdd",
+          "ApiChange": "Resource",
+          "ChangedApiName": "termStore.store,termStore.term,termStore.set,termStore.relation",
```
Comment (Lauragra) [inline changelog/Microsoft.SharePoint.json:8]: You're going to need one ChangeList item for each API/Target. Those fields are single valued.

**Comment by Lauragra** [comment]
@vishriv , please include an update to the toc.yml file in the v1.0 folder, to surface the APIs in the TOC.

File: changelog/Microsoft.SharePoint.json (hunk 1 above)
Comment (vishriv) [reply changelog/Microsoft.SharePoint.json]: Done

**Comment by vishriv** [comment]
> @vishriv , please include an update to the toc.yml file in the v1.0 folder, to surface the APIs in the TOC.

@Lauragra  updated

File: api-reference/v1.0/api/termstore-term-list-relations.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+``` http
+GET https://graph.microsoft.com/v1.0/sites/microsoft.sharepoint.com,b9b0bc03-cbc4-40d2-aba9-2c9dd9821ddf,6a742cee-9216-4db5-8046-13a595684e74/termStore/sets/874be29c-35c8-4295-b3f8-ca2400c966c3/relations
+```
+
+### Response
+>**Note:** The response object shown here might be shortened for readability.
+<!-- {
+  "blockType": "response",
+  "truncated": true,
+  "@odata.type": "collection(microsoft.graph.termStore.relation)"
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-list-relations.md:69]: Please, replace "@odata.type": "collection(microsoft.graph.termStore.relation)" with "@odata.type": "microsoft.graph.termStore.relation" and add the property: "isCollection": true.

File: api-reference/v1.0/api/termstore-term-post.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [term](../resources/termstore-term.md) object.
+
+The following table shows the properties that are required when you create the [term](../resources/termstore-term.md).
+
+|Property|Type|Description|
+|:---|:---|:---|
+|labels|[microsoft.graph.termStore.localizedLabel](../resources/termstore-localizedlabel.md) collection|Label for the term to be created|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-post.md:49]: Please, add a period at the end of the sentence.

File: api-reference/v1.0/api/termstore-term-post.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [term](../resources/termstore-term.md) object.
+
+The following table shows the properties that are required when you create the [term](../resources/termstore-term.md).
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-post.md:45]: For consistency, I recommend you to add object after term. Suggested revision: 

- ...create the \[term\]\(../resources/termstore-term.md\) object.

File: api-reference/v1.0/api/termstore-term-update.md [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+## Request headers
+|Name|Description|
+|:---|:---|
+|Authorization|Bearer {token}. Required.|
+|Content-Type|application/json. Required.|
+
+## Request body
+In the request body, supply a JSON representation of the [term](../resources/termstore-term.md) object.
+
+The following table shows the properties that can be updated for a [term](../resources/termstore-term.md).
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-update.md:44]: For consistency, I recommend you to add object after term. Suggested revision: 

- ...updated for a \[term](../resources/termstore-term.md\) object.

File: api-reference/v1.0/api/termstore-term-update.md [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+
+|Property|Type|Description|
+|:---|:---|:---|
+|labels|[microsoft.graph.termStore.localizedLabel](../resources/termstore-localizedlabel.md) collection|labels of a term|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-update.md:48]: Please,
- capitalize the first letter of a sentence
- add a period at the end of the sentence

File: api-reference/v1.0/api/termstore-term-update.md [hunk 7]
```diff
@@ -0,0 +1,121 @@
...
+|descriptions|[microsoft.graph.termStore.localizedDescription](../resources/termstore-localizeddescription.md) collection|description about the term|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-update.md:49]: Please,
- capitalize the first letter of a sentence
- add a period at the end of the sentence

File: api-reference/v1.0/api/termstore-term-update.md [hunk 8]
```diff
@@ -0,0 +1,121 @@
...
+|properties|[microsoft.graph.keyValue](../resources/keyvalue.md) collection|properties associated with the term|
```
Comment (JarbasHorst) [inline api-reference/v1.0/api/termstore-term-update.md:50]: Please,
- capitalize the first letter of a sentence
- add a period at the end of the sentence

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/api/termstore-relation-post.md [hunk 9]
```diff
@@ -0,0 +1,124 @@
...
+
+
+## Response
+
+If successful, this method returns a `201 Created` response code and a [relation](../resources/termstore-relation.md) object in the response body.
+
+## Examples
+
+### Request
+
```
[the remark this answers is not in the document]
Comment (vishriv) [reply api-reference/v1.0/api/termstore-relation-post.md:61]: @JarbasHorst Here I added 


But the build is failing now :
https://dev.azure.com/msrobots/apidoctor/_build/results?buildId=61608&view=logs&j=fd490c07-0b22-5182-fac9-6d67fe1e939b&t=dac0b392-2308-5704-b9f7-c5b20adf012e&l=12085

I have tried various strings in name, but couldn't find any success. Can you help me out here?
Resolved all other comments

File: changelog/Microsoft.SharePoint.json [hunk 10]
```diff
@@ -1,5 +1,55 @@
...
+          "ChangeType": "Addition",
+          "Description": "Introduced the taxonomy API, including the [termStore.relation](https://docs.microsoft.com/en-us/graph/api/resources/termstore-relation?view=graph-rest-1.0) resource and associated methods.",
+          "Target": "termStore.relation"
+        }
+      ],
+      "Id": "f008d427-4aa4-452d-aa93-0c5c623fdfdd",
+      "Cloud": "prd",
+      "Version": "v1.0",
+      "CreatedDateTime": "2021-07-13T00:00:00",
+      "WorkloadArea": "Sites and lists ",
```
Comment (JarbasHorst) [inline changelog/Microsoft.SharePoint.json:50]: Please, remove the empty space after lists.

File: api-reference/v1.0/api/termstore-relation-post.md (hunk 9 above)
[the remark this answers is not in the document]
Comment (JarbasHorst) [reply api-reference/v1.0/api/termstore-relation-post.md:61]: Thanks for your changes. I'll check internally how we can handle the build issue and get back to you.

**Review by JarbasHorst — CHANGES_REQUESTED** [review]

File: api-reference/v1.0/resources/enums-termstore.md [hunk 11]
```diff
@@ -0,0 +1,40 @@
...
+|Member|
+|:---|
+|global|
+|system|
+|siteCollection|
+
+<!--
+{
+  "type": "#page.annotation",
+  "namespace": "microsoft.graph.termstore"
```
Comment (millicentachieng) [inline api-reference/v1.0/resources/enums-termstore.md:36]: Replace `termstore` with `termStore`

File: api-reference/v1.0/resources/termstore-relation.md [hunk 12]
```diff
@@ -0,0 +1,82 @@
...
+In a pin relationship, a term can be pinned under a different term in a different term set. In a pinned relationship, new children to the term can only be added in the term set in which the term was created. Any change in the hierarchy under the term is reflected across the sets in which the term was pinned. 
+
+The reuse relationship is similar to the pinned relationship except that changes to the reused term can be made from any hierarchy in which the term is reused. Also, a change in hierarchy made to the reused term does not get reflected in the other term sets in which the term is reused.
+
+Inherits from [entity](../resources/entity.md).
+
+## Methods
+|Method|Return type|Description|
+|:---|:---|:---|
+|[List relations](../api/termstore-term-list-relations.md)|[microsoft.graph.termstore.relation](../resources/termstore-relation.md) collection|Retrieve a list of **relation** objects.|
```
Comment (millicentachieng) [inline api-reference/v1.0/resources/termstore-relation.md:25]: Replace `termstore` with `termStore`

File: api-reference/v1.0/resources/termstore-relation.md [hunk 13]
```diff
@@ -0,0 +1,82 @@
...
+|[Create relation](../api/termstore-relation-post.md)|[microsoft.graph.termstore.relation](../resources/termstore-relation.md)|Create a new **relation** object.|
```
Comment (millicentachieng) [inline api-reference/v1.0/resources/termstore-relation.md:26]: Replace `termstore` with `termStore`

Comment (vishriv) [reply api-reference/v1.0/resources/termstore-relation.md]: Done

File: api-reference/v1.0/resources/termstore-relation.md (hunk 12 above)
Comment (vishriv) [reply api-reference/v1.0/resources/termstore-relation.md]: Done

File: api-reference/v1.0/resources/enums-termstore.md (hunk 11 above)
Comment (vishriv) [reply api-reference/v1.0/resources/enums-termstore.md]: Done

**Comment by vishriv** [comment]
@JarbasHorst @Lauragra @millicentachieng Thanks for reviewing this PR. I have worked on and resolved all the comments.

File: api-reference/v1.0/api/termstore-group-delete.md [hunk 14]
```diff
@@ -0,0 +1,87 @@
+---
+title: "Delete termStore group"
```
Comment (Lauragra) [inline api-reference/v1.0/api/termstore-group-delete.md:2]: Global: I see that we're including the subnamespace in the topic title for the v1.0 files, but we didn't do the same in beta, i.e.: https://docs.microsoft.com/en-us/graph/api/termstore-group-delete?view=graph-rest-beta&tabs=http. Let's keep them consistent and remove the subnamespace from the titles in the v1.0 files.

File: api-reference/v1.0/toc.yml [hunk 15]
```diff
@@ -5249,6 +5249,62 @@ items:
           href: api/listitem-list-versions.md
         - name: Get item analytics
           href: api/itemanalytics-get.md
+      - name: Taxonomy
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:5252]: Can you please change ListItems to List items on line 5237? That's an inconsistency I noticed on the site (not related to this PR). Related to this PR, now that the API is GA, we also need to remove "(preview)' from the entry in the beta TOC.

File: api-reference/v1.0/toc.yml [hunk 16]
```diff
@@ -5249,6 +5249,62 @@ items:
...
+        - name: Group
+          href: resources/termstore-group.md
+          items:
+          - name: Create
+            href: api/termstore-group-post.md
+          - name: Get
+            href: api/termstore-group-get.md
+          - name: Delete
+            href: api/termstore-group-delete.md
+          - name: List
```
Comment (Lauragra) [inline api-reference/v1.0/toc.yml:5271]: We should list the CRUD operations in a consistent order in the TOC: List, Create, Get, Update, Delete. Please move List to come before Create.

File: api-reference/v1.0/api/termstore-group-delete.md (hunk 14 above)
Comment (vishriv) [reply api-reference/v1.0/api/termstore-group-delete.md:2]: That was causing issue. Some other file also has the same title and heading 'delete groups', and hence I changed it to complete name. I also changed the title and heading in the corresponding beta file in this PR now for consistency

**Review by Lauragra — APPROVED** [review]

**Pull request merged**